assets/scss/main.scss

```diff
@@ -7,7 +7,7 @@ body {
 
 // link underlines were showing on iOS
 a {
```

Comment:
Maybe add some more spacing making it easier to press the correct link on mobile? A simple mb-2 or mb-4 should do, I think